R/Features-class.R

```diff
@@ -214,8 +214,17 @@ setMethod("show", "Features",
 ##' @exportMethod [
 setMethod("[", c("Features", "ANY", "ANY", "ANY"),
           function(x, i, j, ..., drop = TRUE) {
+              ## Subset the assays
               ans <- callNextMethod(x, i, j, ..., drop)
+              ## Subset the AssayLinks
               ans@assayLinks <- ans@assayLinks[names(ans)]
+              ## Removed lost links
+              allist <- lapply(ans@assayLinks, function(al) {
+                  if(!al@from %in% names(ans)) al <- AssayLink(name = al@name)
```

Comment:
Space between `if (`! :-)

Reply:
Also, no need to assign `al` and then return is immediately. Clearer to return `AssayLink()` right away.

Reply:
Right ! Is it sufficient to put only the `if` statement within the function or should I also explicit the return value if the condition is not met, e.i.:
```r
if (!al@from %in% names(al)) {
    AssayLink(name = al@name)
} else {
    al
}
```

Reply:
Sorry, I misread the code. It is correct as it is.